rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/controllers/BakeryController.groovy

```diff
@@ -176,10 +174,11 @@ class BakeryController {
         def existingBakeStatus = queryExistingBakes(bakeKey)
 
         if (existingBakeStatus) {
-          Id bakesId = registry.createId('bakes').withTag("duplicate", "true")
-          registry.counter(bakesId).increment()
+          registry.counter(registry.createId("bakesRequested", ["flavor":"duplicate"])).increment()
 
           return existingBakeStatus
+        } else {
+          registry.counter(registry.createId("bakesRequested", ["flavor":"plain"]))
```

Comment:
"new" sounds like "rebake" (as in I want a new bake no matter what). I'd call "rebake" "force" except that you call it rebake so think that terminology might be used in practice and didnt want to introduce new terms.

I could use default, though default is even less descriptive. I was thinking "plain" because the request isnt decorated (not rebake). "duplicates" happen to be a variant of "plain" though that isnt evident. And "plain" and "flavor" went together. 

What about "normal"?